ros2_controllers/doc/index.rst

```diff
@@ -0,0 +1,62 @@
+Welcome to the documentation for ros2_controllers
+=================================================
+
+For more information of the ros2_control framework see `control.ros.org <https://control.ros.org/>`__.
+
+.. list-table::
```

Comment:
you mean linking to the respective controller-docs on control.ros.org? I could do that.

Reply:
I'm not sure if one can (already) configure rosdoc2 to configure for custom builders which need other dependencies (like generate_parameter_library in this case). And we said that we want to avoid duplicating information ;)

Reply:
I added the links, see screenshot above

Reply:
what do you think @fmauch ?

Reply:
no problem, thanks :)